modules/administration/pages/content-lifecycle-examples.adoc

```diff
@@ -0,0 +1,260 @@
+[[content-lifecycle]]
+= Content Lifecycle Management Examples
+
+This section contains some common examples of how you can use content lifecycle management.
+Use these examples to build your own personalised implementation.
+
+
+
+[#create-proj-for-patch-cycle]
+== Creating a Project for a Monthly Patch Cycle
+
+
+
+* Creating a filter for a project
+* Adding a filter to a project
+* Applying the filter to a new build environment
+* Excluding a patch from a project
+* Including a patch in a project
+
+
+
+[#create-a-filter]
+=== Create a Filter for the Project
```

Comment:
```suggestion
=== Creating a Filter for the Project
```
(For consistency.  Lana will probably propose to get rid if the ing form in section titles.)

Reply:
Yeah, base form of the verb in section titles, gerund form in procedure titles. Sorry!